caesar_cipher/lib/cipher.rb

```diff
@@ -0,0 +1,29 @@
+class Cipher
+
+  def initialize(rot_key, raw_message)
+    @rot_key = rot_key
+    @raw_message = raw_message
+  end
+
+  def encrypt
+
+    encrypted_message = ''
+    @raw_message.each_char do |character|
+      character = character.ord
```

Comment:
@zzokki81 try not to modify the iteration variable within the iteration. Ruby will handle it, but it's confusing changing it with the loop for other programmers. Use different local variable if you need to change it.